core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildLocalRunner.java

```diff
@@ -17,10 +15,8 @@ public NativeImageBuildLocalRunner(String nativeImageExecutable) {
 
     @Override
     public void cleanupServer(File outputDir, boolean processInheritIODisabled) throws InterruptedException, IOException {
-        final ProcessBuilder pb = new ProcessBuilder(nativeImageExecutable, "--server-shutdown");
-        pb.directory(outputDir);
-        final Process process = ProcessUtil.launchProcess(pb, processInheritIODisabled);
-        process.waitFor();
+        final String[] cleanupCommand = { nativeImageExecutable, "--server-shutdown" };
+        runCommand(cleanupCommand, null, outputDir);
```

Comment:
It doesn't seem like this takes `processInheritIODisabled` does it?

Reply:
You are right, this slipped away. I never used native-image server so I am not sure if `processInheritIODisabled` is needed here. If so, I'll bring it back.

Reply:
So `--server-shutdown` prints something like:
```
Cleanup Server(pid: 1978480, port: 33031) running for: 00:00:40
```

TBH I don't see why we would want this logged.

Reply:
Yeah, it does seem totally useless